controllers/componentversion_contoller_test.go

```diff
@@ -189,13 +190,6 @@ func TestComponentVersionSemverCheck(t *testing.T) {
 			latestVersion:     "0.0.2",
 			expectedUpdate:    true,
 		},
-		{
```

Comment:
Heh, this never worked because I missed that I didn't change the latestVersion field.